src/crdt.rs

```diff
@@ -1370,12 +1394,25 @@ mod tests {
         assert_eq!(d.version, 0);
         let mut crdt = Crdt::new(d.clone()).unwrap();
         assert_eq!(crdt.table[&d.id].version, 0);
+        assert!(crdt.alive.get(&d.id).is_none());
```

Comment:
`get(x).is_none()` -> `!contains()`  x 3

Reply:
✔️  Looks like it's called [contains_key](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.contains_key)